tests/mock_network/churn.rs

```diff
@@ -560,3 +560,33 @@ fn messages_during_churn() {
         verify_invariant_for_all_nodes(&network, &mut nodes);
     }
 }
+
+#[test]
+fn remove_unresponsive_node() {
+    let min_section_size = 4;
+    let network = Network::new(min_section_size, None);
+    let mut nodes = create_connected_nodes(&network, 2 * min_section_size);
```

Comment:
Why `2 * min_section_size`?

Reply:
just follow the other tests make a network of 8